drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -95,6 +100,43 @@ def ssn_serial
     self.SSN&.[](-4..-1)
   end
 
+  def mci_external_id
+    external_ids_by_slug('clientview').first
+  end
+
+  def warehouse_id
+    destination_client.id
+  end
+
+  def warehouse_url
+    "https://#{ENV['FQDN']}/clients/#{id}/from_source"
+  end
+
+  def mci_url
+    return unless mci_external_id&.remote_credential&.present?
+
+    "#{mci_external_id.remote_credential.link_base}/ClientInformation/Profile/#{mci_external_id.value}?aid=2"
+  end
+
+  def external_identifiers
```

Comment:
Let's add the `label` field directly in here, instead of using titleize. The MCI ID is appearing in the UI as `Mci ID` which is not the right casing. Also, the client_id label in the UI should be `HMIS ID`.